test/SlowTests/Server/Documents/PeriodicBackup/PeriodicBackupSlowTests.cs

```diff
@@ -3482,38 +3482,55 @@ public async Task NumberOfCurrentlyRunningBackupsShouldBeCorrectAfterBackupTaskD
 
             var backupPath = NewDataPath(suffix: "BackupFolder");
 
-            using (var store = GetDocumentStore())
+            using (var server = GetNewServer(new ServerCreationOptions
+            {
+                CustomSettings = new Dictionary<string, string>
+                {
+                    [RavenConfiguration.GetKey(x => x.Backup.MaxNumberOfConcurrentBackups)] = 1.ToString()
+                }
+            }))
+            using (var store = GetDocumentStore(new Options
```

Comment:
Can we extend this to include it when `DatabaseMode` is `RavenDatabaseMode.Sharded`?

```
[RavenTheory(RavenTestCategory.Smuggler)]
[RavenData(DatabaseMode = RavenDatabaseMode.All)]
public async Task NumberOfCurrentlyRunningBackupsShouldBeCorrectAfterBackupTaskDelay(Options options)
{
...
}
```

Reply:
Please handle that in separate PR. I'm merging this one